docs/ref/csrf.txt

```diff
@@ -265,6 +269,12 @@ dangerous side effects , and so a CSRF attack with a GET request ought to be
 harmless. :rfc:`2616` defines POST, PUT and DELETE as 'unsafe', and all other
 methods are assumed to be unsafe, for maximum protection.
```

Comment:
This reads wrong... "X,Y,Z are unsafe, and all others are assumed to be  unsafe" -- I think the last word should be "safe" (I know, it isn't part of the current PR, but while we're here, let's fix this too).